testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -171,6 +171,7 @@ def test_Universe_invalidpermissionfile_IE_msg(self):
         try:
             mda.Universe(os.path.join(temp_dir.name, 'permission.denied.tpr'))
         except IOError as e:
+            #pylint: ignore=unsupported-membership-test
```

Comment:
Could there be a difference between the version of pylint you use locally and the one on travis?